### DISCUSSION TITLE
Is dependency track Helm chart production ready?

### DISCUSSION THREAD

**Discussion by Yo150850** [DependencyTrack/dependency-track#3836] [General]
As "https://github.com/evryfs/helm-charts/blob/master/charts/dependency-track" is no longer maintained, any near future plan to make "https://github.com/DependencyTrack/helm-charts" production ready? From database point of view, would postgreSQL be incorporated like we had in everyfs' helm chart?

**Comment by nscuro** [comment]
> any near future plan to make "https://github.com/DependencyTrack/helm-charts" production ready?

It **is** production ready in the sense that we are not aware of any major outstanding issues that need doing.

> From database point of view, would postgreSQL be incorporated like we had in everyfs' helm chart?

It will not.

**Reply by Yo150850** [reply]
Ok, thanks for information. Any specific reason why postgresSQL won't be incorporated?
